### ISSUE TITLE
Unlimited Restricted Airspace Displayed as FLNaN

### ISSUE THREAD

**Issue by Fechulo** [zlsa/atc#756]
When the ceiling of restricted airspace is unlimited, it is displayed as FLNaN in the game, instead of UNL. It can be seen at UUDD:
![flnan](https://cloud.githubusercontent.com/assets/13125311/21119863/65cc0b4e-c0bc-11e6-9483-ac74a55c857d.PNG)

**Comment by n8rzz** [comment]
@Fechulo @erikquinn That is because it's trying to parse a string 'Unlimited' which is obviously, not a number.  

This should, hopefully, be an easy fix.  Should it display `FL UNL` or just `UNL`?

**Comment by n8rzz** [comment]
never mind, I found it.

**Comment by n8rzz** [comment]
@Fechulo is this the first you noticed this issue?  From what I can tell, this much have been an issue for quite a while.  Oh well, it's going to be addressed for v3.2.0.

**Comment by Fechulo** [comment]
@n8rzz I believe it should just display 'UNL'. This was definitely working before v3 but I'm not sure when it stopped working. There's very few airports in the game that have restricted airspace with an unlimited ceiling, so I found it today while working on El Paso Intl.

**Comment by n8rzz** [comment]
@Fechulo right on, I think there were two airports that came up when I searched.  I'm sure I got a little jumpy refactoring and something was missed.  In fact, thats my assumption here.  

Good news is that there are now specific tests for this case, so once the bugfix is completed we will know that this case is covered with tests.

**Comment by erikquinn** [comment]
@n8rzz does `Infinity` parse properly as a number? Perhaps using that in the file instead of `unlimited` would yield an equally acceptable result without necessitating complications to the logic for this one case? Either way. I do agree with @Fechulo though, definitely `UNL` of the two or `999` or `600` (the top of controlled airspace is more of less considered to be FL600, where the jet airways are no longer usable).

**Comment by n8rzz** [comment]
@erikquinn `parseFloat(Infinity)` will output `Infinity`.  The problem here, is that infinity is being passed as a string and not a number (Infinity _is_ a number in javascript):
```javascript
parseFloat(Infinity) -> Infinity
parseFloat('Infinity') -> NaN
```

As far the FL and the UNL, that is already implemented within the `CanvasController` but because the height value is `NaN` by the time it gets there, the ternary thinks is supposed to be a `FL`.
```javascript
const height = (area.height === Infinity ? 'UNL' : 'FL' + Math.ceil(area.height / 1000) * 10);
```

**Comment by erikquinn** [comment]
Resolved by #755. Closing.
